jdaviz/configs/default/plugins/viewers.py

```diff
@@ -221,3 +221,7 @@ def jdaviz_helper(self):
     @property
     def reference_id(self):
         return self._reference_id
+
+    @property
+    def reference(self):
```

Comment:
Wut? How is this different from reference ID? I wasn't aware this even existed.

Reply:
ugh, I know, but they're subtly different for Imviz.  I think we could definitely use a full code-audit to make sure we are using and naming all of these consistently...